packages/auth/src/auth.decorators.ts

```diff
@@ -7,8 +7,8 @@ import {
   UseGuards,
 } from '@nestjs/common';
 import { Session } from '@shopify/shopify-api';
-import type { IncomingMessage } from 'http';
 import { FastifyRequest } from 'fastify';
+import type { IncomingMessage } from 'node:http';
```

Comment:
`IncomingMessage` is not used in this file?

Reply:
it used line 31
`const req = ctx
    .switchToHttp()
    .getRequest<ShopifySessionRequest<IncomingMessage | FastifyRequest>>();`